src/main/sql/delete_rls_role.sql

```diff
@@ -0,0 +1,22 @@
+--/
+CREATE OR REPLACE SCRIPT DELETE_RLS_ROLE(role_name) AS
+import('"'..exa.meta.script_schema..'".EXA_RLS_BASE', 'BASE')
+_role_id = BASE.get_role_id_by_name(role_name)
+if _role_id==nil then error('No such role_name: "'..role_name) end
+-- Remove role from EXA_ROLES_MAPPING table
+query([[DELETE FROM ::s.EXA_ROLES_MAPPING WHERE role_id = :i]],
+      {s = exa.meta.script_schema, i = _role_id })
+-- Remove role from all users
+query([[UPDATE ::s.EXA_RLS_USERS SET EXA_ROLE_MASK = EXA_ROLE_MASK - POWER(2,:i-1)
```

Comment:
This POWER is an SQL function, you cannot use the functions from the base module there. And this function is too expensive because it makes a lookup in the roles table. The role_id is already given.
Minus will be replaced by BIT_AND(EXA_ROLE_MASK,BIT_NOT(POWER(2,:i-1)))